packages/schema/bind/src/bindings/wrap-bindgen.ts

```diff
@@ -0,0 +1,70 @@
+import { BindOutput, BindOptions } from "../types";
+import { Args_generateBindings, GenerateBindingFn, Output } from "./types";
+
+import {
+  PolywrapClient,
+  PolywrapClientConfigBuilder,
+  ExtendableUriResolver,
+} from "@polywrap/client-js";
+import { OutputEntry } from "@polywrap/os-js";
+
+export function getGenerateBindingFn(uri: string): GenerateBindingFn {
+  return (options: BindOptions) => invokeBindgenWrap(uri, options);
+}
+
+async function invokeBindgenWrap(
+  uri: string,
+  options: BindOptions
+): Promise<BindOutput> {
+  // TODO: Can simplify if github resolver is added to a default config bundle
+  const config = new PolywrapClientConfigBuilder()
+    .addInterfaceImplementation(
+      ExtendableUriResolver.defaultExtInterfaceUris[0].uri,
+      "wrap://ipfs/QmYPp2bQpRxR7WCoiAgWsWoiQzqxyFdqWxp1i65VW8wNv2"
+    )
+    .addDefaults()
+    .build();
+
+  const client = new PolywrapClient(config);
+  const args: Args_generateBindings = {
+    wrapInfo: {
+      version: options.wrapInfo.version,
+      name: options.wrapInfo.name,
+      type: options.wrapInfo.type,
+      abi: JSON.stringify(options.wrapInfo.abi),
+    },
+    context: options.config ? JSON.stringify(options.config) : undefined,
+  };
+
+  const result = await client.invoke<Output>({
+    uri,
+    method: "generateBindings",
+    args: (args as unknown) as Record<string, unknown>,
```

Comment:
Unrelated: Is this casting really required? If so I think we should fix this because it isn't a very good developer experience.

Reply:
Yeah, it's required when using statically typed args objects. TypeScript complains without the cast because `args` is of type `Record<string, unknown> | Uint8Array | undefined`.